Foundation/NSURL.swift

```diff
@@ -610,8 +610,28 @@ open class NSURL : NSObject, NSSecureCoding, NSCopying {
     */
     /// - Experiment: This is a draft API currently under consideration for official import into Foundation as a suitable alternative
     /// - Note: Since this API is under consideration it may be either removed or revised in the near future
-    open func checkResourceIsReachable() throws -> Bool {
-        NSUnimplemented()
+    open func checkResourceIsReachableAndReturnError(_ error: UnsafeMutablePointer<NSError?>?) -> Bool {
```

Comment:
Yes, I did it for compatibility with Darwin. I didn't know about the issue (i guess i've just missed it). I'll revert this change.